keyRegistration/js/registration.js

```diff
@@ -155,21 +155,21 @@ $(document).ready(function()
         $('#key_label').removeClass("error");
     }
 
-    function requiredFieldsSet(){
+    function requiredFieldsSet() {
```

Comment:
Opening brace should be on a new line

*PHPCodeSnifferBear, severity NORMAL, section `js`.*